src/main.cpp

```diff
@@ -248,23 +248,24 @@ class BroadcastIPTask : public AbstractTask<10 * configMINIMAL_STACK_SIZE> {
     }
 };
 
+
 void app_main(void) {
     IBSP* bsp = &BspContainer::getBSP();
     bsp->initChip();
     INetworkManager* networkManager = &NetworkContainer::getNetworkManager();
     networkManager->start();
 
-    static HiveMindMessageSender s_spiMessageSend("hivemind_send", tskIDLE_PRIORITY + 1);
-    static HiveMindDispatcher s_spiDispatch("hivemind_receive", tskIDLE_PRIORITY + 1);
+    static HiveMindMessageSender s_spiMessageSend("hivemind_send", tskIDLE_PRIORITY + 10);
```

Comment:
We could maybe extract these priorities to constants so they are easier to change need be.